docs/make_projection.py

```diff
@@ -42,7 +42,8 @@
 PRJ_SORT_ORDER = {'PlateCarree': 1,
                   'Mercator': 2, 'Mollweide': 2, 'Robinson': 2,
                   'TransverseMercator': 2, 'LambertCylindrical': 2,
-                  'LambertConformal': 2, 'Stereographic': 2, 'Miller': 2,
```

Comment:
It may be that you actually need to *run* this script and commit the output (I forget).

We should probably write up a checklist for creating a new projection...